examples/traits/prometheus/README.md

```diff
@@ -0,0 +1,19 @@
+# Camel K Prometheus Trait
+
+In this section you will find examples about fine tuning your `Integration` using **Prometheus** `trait` capability.
+
+## Use Case Demonstration
+
+The following query may be used to get statistics about the number of events successfully handled by the `Integration`
+    
+```
+    public void configure() {
+        from("timer:stream?period={{timer.period}}")
+            .routeId("unreliable-service")
+            .setBody(header(Exchange.TIMER_COUNTER).prepend("event #"))
+            .to("microprofile-metrics:meter:success"); 
+    }
+```
+
+Similarly other use cases can be to retrieve information on unprocessed events, number of retries made to process an event, etc. For more information on configuring custom or native Camel K Integration metrics, please refer to the [6.2. Configuring Camel K integration metrics](https://access.redhat.com/documentation/en-us/red_hat_integration/2020-q4/html/deploying_camel_k_integrations_on_openshift/monitoring-camel-k) section. 

```

Comment:
We are a vendor-neutral opensource project, so it's better to avoid referring a specific product's documentation.
Instead, we have this documentation for monitoring Camel K:
https://camel.apache.org/camel-k/1.8.x/observability/monitoring/integration.html